### ISSUE TITLE
False positive result in 0.800: "run_path" does not return a value

### ISSUE THREAD

**Issue by yanqd0** [python/mypy#9996]
**Bug Report**

There is a false positive result in the latest version `0.800`, which not exist in `0.790`.

**To Reproduce**

Prepare a `demo.py` file like this:

```python
import runpy

info = runpy.run_path('hello.py')
print(info)
```

**Expected Behavior**

In the version `0.790`:

```shell
$ mypy demo.py
Success: no issues found in 1 source file
```

**Actual Behavior**

In the version `0.800`:

```shell
$ mypy demo.py
demo.py:3: error: "run_path" does not return a value
Found 1 error in 1 file (checked 1 source file)
```

**Your Environment**

Python version: `3.8.5`

**Comment by hauntsaninja** [comment]
Thanks, already fixed in https://github.com/python/typeshed/pull/4965/files

**Comment by yanqd0** [comment]
It is still not working in `mypy-0.812`. When `typed-ast` is fixed to `1.4.2`, `mypy-0.790` is good, but `mypy-0.812` will still report `error: "run_path" does not return a value`.
